.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: rust
 
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - os: linux
+    - os: osx
+      env: CC=gcc-9 FC=gfortran-9 LIBRARY_PATH=/usr/local/opt/gcc/lib/gcc/9
```

Comment:
Does it mean we now introduce some new requirements to the usage of this crate? I mean those environment variables were not needed before. Why does Brew not switch to 9 by default?

Reply:
`gcc-9` is the default `gcc` in Brew _at the moment_.
But then there are two moments:
* Brew can switch to a newer version any time without notice (thus we install `gcc@9` formula and not just `gcc`)
* Default `gcc` in `PATH` on macOS points to `/usr/bin/gcc` which in turn points to system `clang` (even when you install `gcc` formula from Brew, if you type `gcc` in console, it will just call `clang`). Thus I use env variables to select both C and Fortran compilers from the same distribution

Reply:
Good points!